.github/workflows/go.yml

```diff
@@ -35,6 +35,10 @@ jobs:
         env:
           GO111MODULE: on
         run: |
+          wget https://dl.min.io/server/minio/release/linux-amd64/minio && chmod +x minio
+          mkdir -p ~/.minio/certs/ && openssl req -x509 -newkey rsa:1024 -keyout ~/.minio/certs/private.key -out ~/.minio/certs/public.crt -days 7 -nodes -subj '/CN=localhost'
```

Comment:
You can get the certs from minio-go repo if necessary for localhost keys @vadmeste Or commit the certs into mc repo.

Reply:
Or better use mkcert in batch mode to generate and trust the certs